LICENSE.md

```diff
@@ -1,4 +1,4 @@
-Shotgun engine for Unity copyright © 2019 Unity Technologies ApS
+Shotgrid engine for Unity copyright © 2019 Unity Technologies ApS
```

Comment:
The date is interesting here, since Shotgrid did not exist back in 2019.
Please ask @benoithudson what to do for this one. I believe we generally want the year when the file first appeared, which is correct here
I might be confused, I handled copyright years multiple ways on multiple code bases